app/src/componentLibrary/inputNumeric/inputNumeric.scss

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.container {
+  position: relative;
+  display: inline-block;
+
+  &.disabled {
+    opacity: 0.3;
+  }
+}
+
+.label {
+  margin-bottom: 4px;
+  font-family: $FONT-ROBOTO-MEDIUM;
+  font-size: 13px;
+  line-height: 20px;
+  text-indent: 1px;
+  color: $COLOR--almost-black;
+}
+
+.sign {
+  width: 16px;
+  height: 16px;
```

Comment:
Please add `display: inline-block` to be able to use size properties for inline element.